internal/strace/strace.go

```diff
@@ -186,8 +190,20 @@ func (r *Result) parseEnterSyscall(syscall, args string) error {
 		if err != nil {
 			return err
 		}
+		writeBuffer := ""
 		match := writePattern.FindStringSubmatch(args)
-		r.recordFileWrite(match[1], bytesWritten)
+		firstQuoteIndex := strings.Index(args, "\"")
+		lastQuoteIndex := strings.LastIndex(args, "\"")
+		if firstQuoteIndex != -1 && lastQuoteIndex != -1 && lastQuoteIndex > firstQuoteIndex {
+			// Save the contents between the first and last quote.
+			writeBuffer = args[firstQuoteIndex+1 : lastQuoteIndex]
+			lastIndexComma := strings.LastIndex(args, ",")
+			if lastQuoteIndex+1 != lastIndexComma {
+				// If the write buffer is truncated there will be an ellipses after the quoted write buffer. Save this as well.
```

Comment:
Do we want to save the ellipses of truncated writes or represent that the file write is truncated in another way?

Reply:
can it be recorded in the main JSON file? I personally don't think we should be saving the ellipses since they're an artifact of the gVisor logs

Reply:
Yeah, I was considering this,. Originally I was wondering if we needed that if the size to be truncated was fixed, but since you mentioned it can be configurable (and in the case where the underlying strace representation changes) it could be good to have that boolean flag. I'll put up a change for this.

Reply:
Given that we have the write size, do we need this flag/indication at all?

Reply:
That's true. I think it would be good to communicate somewhere that the write buffers may not contain all of the write contents though. Is there a good place to do that?

Reply:
I think that's OK for now. Users will be able to tell by comparing the actual size of the contents vs the reported bytes written. 

If the need arises for an explicit flag, we can always add it. Removing it is harder.

Reply:
Good point that removing a field is harder than adding one

Reply:
Sg, removed the ellipses.